core/src/main/java/com/klaytn/caver/tx/kct/KIP7Token.java

```diff
@@ -0,0 +1,776 @@
+package com.klaytn.caver.tx.kct;
+
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.crypto.KlayCredentials;
+import com.klaytn.caver.methods.response.KlayLogs;
+import com.klaytn.caver.methods.response.KlayTransactionReceipt;
+import com.klaytn.caver.tx.SmartContract;
+import com.klaytn.caver.tx.manager.TransactionManager;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import org.web3j.abi.FunctionEncoder;
+import org.web3j.abi.TypeReference;
+import org.web3j.abi.datatypes.Address;
+import org.web3j.abi.datatypes.Bool;
+import org.web3j.abi.datatypes.Event;
+import org.web3j.abi.datatypes.Function;
+import org.web3j.abi.datatypes.Type;
+import org.web3j.abi.datatypes.Utf8String;
+import org.web3j.abi.datatypes.generated.Uint256;
+import org.web3j.abi.datatypes.generated.Uint8;
+import org.web3j.protocol.core.RemoteCall;
+import org.web3j.tx.gas.ContractGasProvider;
+
+
+public class KIP7Token extends SmartContract {
```

Comment:
Do we need `Token` prefix?

Reply:
Because KIP-7 Standard title is "Fungible Token Standard", I named this class "KIP7Token".
However, I agree with you because KIP-7 itself contains the contents of Fungible Token. 
I will fix it as you said.

Reply:
Please make this consistent with caver-js. cc @jimni1222

Reply:
I'll rename Class and file name KIP7